uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/UiAutomatorUtils.java

```diff
@@ -164,6 +173,7 @@ public static UiObject obtainUiObjectWithTextAndClassType(@NonNull final String
      */
     public static UiObject obtainChildInScrollable(@NonNull final String scrollableResourceId,
                                                    @NonNull final String childText) {
+        Logger.i(TAG, "Obtain a child element inside a scrollable view by specifying resource id and text..");
```

Comment:
let's also log the resource id in the log message

Reply:
same feedback for all other instance...when we are looking for specific things via Ui Automator...let's log what we are looking for

Reply:
added.